Bio/bgzf.py

```diff
@@ -463,7 +463,8 @@ def _load_bgzf_block(handle, text_mode=False):
             if block_size is not None:
                 raise ValueError("Two BC subfields?")
             block_size = struct.unpack("<H", subfield_data)[0] + 1  # uint16_t
-    assert x_len == extra_len, (x_len, extra_len)
+    if x_len != extra_len:
+        raise ValueError(f"x_len and extra_len differ {x_len}, {extra_len}")
```

Comment:
The message is essentially for debugging, this would be more elegant:

```python
raise ValueError(f"{x_len=} and {extra_len=} differ")
```

Sadly that needs Python 3.8 onwards.

Reply:
I would have done that if 3.7 wasn't supported any more, but it still is if I am still up to date.

Reply:
No, I was only waiting for the CI to finish before merging your contribution.